helpers/utils.py

```diff
@@ -80,6 +83,31 @@ def download_file(url, direct_access=False, user=None, password=None):
     return tmpfile
 
 
+def send_analysis_result_email(analysis):
+    """Helper function to send email about the analysis result.
+
+    :param analysis: analysis model
+    :type analysis: geosafe.models.Analysis
+    """
+    if settings.EMAIL_ENABLE and analysis.user.email:
+        analysis_url = reverse(
+            'geosafe:analysis-create', kwargs={'pk': analysis.pk})
+        analysis_url = urlparse.urljoin(
+            settings.GEONODE_BASE_URL, analysis_url)
+        subject_email = _("Your GeoSAFE analysis is finished!")
+        message = _("Your GeoSAFE analysis is finished! Visit {0} "
+                    "to see the result.").format(analysis_url)
```

Comment:
Nice @myarjunar 

The best practices for emails is to provide two emails template for each subject type, the first one for plain text, and the other one for html.

See here:
https://stackoverflow.com/questions/2809547/creating-email-templates-with-django